src/style.css

```diff
@@ -1 +1,16 @@
-/* styles go here */
+
+.form-fild {
+  margin-bottom: 10px;
+}
+  
+.form-fild:last-child {
+  margin-bottom: 0;
+}
+
+.form-group {
+  margin-bottom: 20px;
+}
+
+.Additional-info {
```

Comment:
use kebab-case for class naming

Reply:
- [DEMO LINK](https://DmitriyChernega.github.io/layout_html-form/)
- [TEST REPORT LINK](DmitriyChernega.github.io/layout_html-form/report/html_report/)